NtupleProducer/python/HiggsTauTauProducer_94X.py

```diff
@@ -635,17 +637,17 @@
 ### ----------------------------------------------------------------------
 ### Search for FSR candidates
 ### ----------------------------------------------------------------------
-process.load("UFHZZAnalysisRun2.FSRPhotons.fsrPhotons_cff")
-process.appendPhotons = cms.EDProducer("LeptonPhotonMatcher",
-    muonSrc = cms.InputTag("softMuons"),
-    electronSrc = cms.InputTag("cleanSoftElectrons"),
-    photonSrc = cms.InputTag("boostedFsrPhotons"),#cms.InputTag("cmgPhotonSel"),
-    matchFSR = cms.bool(True)
-    )
```

Comment:
why is this discarded? was there a specific reason?

Reply:
just didn't want to download that package because we don't use this (do we?)

Reply:
I don't think we used it so far, but I would keep this, just in case..
(in any case it can be easily disabled from the config)